### DISCUSSION TITLE
Is faucet.rinkeby.io still down for you ?

### DISCUSSION THREAD

**Discussion by oscarmairey** [smartcontractkit/full-blockchain-solidity-course-py#231] [General]
![image](https://user-images.githubusercontent.com/92814105/138165976-39ccd471-02d1-48f0-95dc-315742de3e66.png)

This has been showing for the last couple of days on my side and I dont see no one talking about it

**Comment by cromewar** [comment]
Yeah the page is down, however you can get ETH using chainlink's faucet: https://faucets.chain.link/

**Comment by PatrickAlphaC** [comment]
The rinkeby faucet is having a hard time... do any of these work? https://docs.chain.link/docs/link-token-contracts/#rinkeby
